backend/src/main/java/com/woowacourse/friendogly/auth/dto/KakaoTokenResponse.java

```diff
@@ -0,0 +1,11 @@
+package com.woowacourse.friendogly.auth.dto;
+
+public record KakaoTokenResponse(
+        String token_type,
```

Comment:
```suggestion
package com.woowacourse.friendogly.auth.dto;

public record KakaoTokenResponse(
        @JsonProperty(value = "token_type")
        String tokenType,
```

카카오 명세 때문에 snake_case를 쓴 것 같아요.
JsonProperty 어노테이션을 사용하면 어떨까요?